lib/node_modules/@stdlib/repl/test/integration/test.auto_close_pairs.js

```diff
@@ -166,8 +166,6 @@ function moveBack( stream, N ) {
 * @private
 * @param {Object} t - test object
 * @param {Object} fixture - fixture object
-* @param {string} fixture.expression - incomplete expression string
```

Comment:
@Snehil-Shah These should not have been removed. This is how we document required/expected properties. It is not enough that `fixture` is an object; in this case, `fixture` must have `expression` and `cursor` properties. We are not completely consistent on this, especially when an input object is expected to have many properties and/or properties/methods may frequently change, but, in this case, it is reasonable to document and leave in.